docs/academy/herocourse/module4.md

```diff
@@ -279,76 +273,88 @@ yarn start:docker
 
 
 
-#### Step 4: Query the project
+#### Step 6: Query the Project
 
-Once the docker container is up and running, which could take a few minutes, open up your browser and navigate to `www.localhost:3000`. 
+Once the docker container is all set and running successfully, which may take a few minutes, open up your browser and navigate to `www.localhost:3000`. 
```

Comment:
Just a suggestion @ZeelPatel1102 Docker being up is a phrase, also related to docker command. I would keep it here, makes more sense to me that "all set"

Reply:
Oh! Thanks for this information. I will update this accordingly. I think I have done the same in some other file as well, I will have a look into it. Thanks a lot.

Reply:
No worries! Happy to help